cloud/audit/v1/LogEntryData.ts

```diff
@@ -498,7 +495,7 @@ export interface DestinationAttributes {
     /**
      * The network port of the peer.
      */
-    port?: number | string;
+    port?: number;
```

Comment:
Does this allow the generated library to surface errors more easily, or is this adding new errors where previously the underlying data structures/APIs would accept a string?

Reply:
This gives a narrower definition of the properties so a user does not need to check if the data is a string or number.

The port is always a number from the API for example.